### ISSUE TITLE
Reproduce Issue: Up-A CIFAR-100, Flowers 102

### ISSUE THREAD

**Issue by dev-sungman** [OpenGVLab/gv-benchmark#6]
Hi, 

When I reproduce the linear-probe classification performance on CIFAR-100, Flowers 102, 
I got weird results when Up-A R50 was used for the backbone. 

In the paper, Up-A R50 outperforms ImageNet pre-trained R50, however, ImageNet pre-trained R50 outperforms Up-A R50 on CIFAR-100, Flowers 102 cases with large margins. 

My configurations as below:
Model
![image](https://user-images.githubusercontent.com/36710497/168071890-d3255a5d-6e34-4d11-8555-bb35c45f9db6.png)

Dataset
![image](https://user-images.githubusercontent.com/36710497/168071952-11103896-82ac-49be-9ec8-ff0f9a21382d.png)
![image](https://user-images.githubusercontent.com/36710497/168071982-785f5da7-9b1f-4a4d-a1bc-382582c01611.png)

Others
![image](https://user-images.githubusercontent.com/36710497/168072016-19d95ce6-5874-4839-9cea-3d2d67c37858.png)

I checked whether the pre-trained Up-A was successfully loaded or not.

Thanks in advance.

**Comment by dev-sungman** [comment]
If you could provide the linear-probe configuration that uses Up-A, it must be helpful to me.

**Comment by yinanhe** [comment]
In linear-probe setting, we select the learning rate and weight decay with a grid search of 3 logarithmically spaced, in which between 1e-4 and 0.1 for learning rate, between 1e-6 and 1e-4 for weight decay, we also searched two momentum 0.9 and 0.99. Besides that, 30 is also a work learning rate here.

**Comment by dev-sungman** [comment]
Thanks very much !
